lib/WGmeta/Cli/Commands/Set.pm

```diff
@@ -0,0 +1,117 @@
+package WGmeta::Cli::Commands::Set;
+use strict;
+use warnings FATAL => 'all';
+use experimental 'signatures';
+
+require WGmeta::Cli::Commands::Command;
+use WGmeta::Wireguard::Wrapper::Config;
+our @ISA = qw(WGmeta::Cli::Commands::Command);
+
+use constant WIREGUARD_HOME => '/home/tobias/Documents/wg-meta/t/Data/';
+use constant TRUE => 1;
+use constant FALSE => 0;
+
+sub new($class, @input_arguments) {
+    my $self = $class->SUPER::new(@input_arguments);
+
+    bless $self, $class;
+
+    return $self;
+}
+
+sub entry_point($self) {
+    if ($self->_retrieve_or_die($self->{input_args}, 0) eq 'help') {
+        $self->cmd_help();
+        return
+    }
+    else {
+        my $wg_home;
+        # check if env var is available
+        if (defined($ENV{'WIREGUARD_HOME'})) {
+            $wg_home = $ENV{'WIREGUARD_HOME'};
+        }
+        else {
+            $wg_home = WIREGUARD_HOME;
+        }
+
+        # would be very nice if we can set a type hint here...possible?
+        $self->{'wg_meta'} = WGmeta::Wireguard::Wrapper::Config->new($wg_home);
+        $self->_run_command();
+    }
+}
+
+sub _run_command($self) {
+    my $interface = $self->_retrieve_or_die($self->{input_args}, 0);
+    my $offset = -1;
+    my $cur_start = 0;
+    my @input_args = @{$self->{input_args}};
+    for my $value (@{$self->{input_args}}) {
+        if ($value eq 'peer') {
+
+            # if there is just one peer we skip here
+            if ($offset != 0) {
+                $self->_apply_change_set($interface, @input_args[$cur_start ... $offset]);
```

Comment:
do you actually mean the "three dot operator" ?

Reply:
Actually not...(meat the `..` operator). Interesting that program worked as expected though..